packages/components/src/button/style.scss

```diff
@@ -285,44 +285,6 @@
 	}
 }
 
-// ==========================================================================
-// Resets
-// ==========================================================================
-
-// Turn Reset 'buttons' into regular text links
-.layout__content input,
-.dialog__content input {
-	&[type='reset'],
-	&[type='reset']:hover,
-	&[type='reset']:active,
-	&[type='reset']:focus {
-		background: 0 0;
-		border: 0;
-		padding: 0 2px 1px;
-		width: auto;
-		box-shadow: none;
-	}
-}
-
-// Buttons within sentences sit on the text baseline.
-.layout__content p .button,
-.dialog__content p .button {
-	vertical-align: baseline;
-}
```

Comment:
How did you confirm this was un-used?

Reply:
I'm not 100% sure, but I spent some considerable effort trying to find usages. Not only today, but also a few months ago when working on the CSS migration. Never found anything.

This style was added in a prehistoric PR that introduced the `<Button>` component for the first time.

Reply:
Sounds good! Even if there are one or two usages we didn't catch, the effect is going to be only visual and fairly minor potentially so not a biggie. It's not a good rule anyway to have at this level.

Reply:
Yes, the rule doesn't even make any sense to me: why should buttons inside `p` be aligned differently? I suspected that the TinyMCE editor or Reader could make use of that, but didn't find any evidence.